docs/docsite/rst/dev_guide/platforms/vmware_guidelines.rst

```diff
@@ -59,12 +59,14 @@ Your NFS server must expose the following directory structure:
     $ tree /srv/share/
     /srv/share/
     ├── isos
-    │   ├── base.iso
     │   ├── centos.iso
     │   └── fedora.iso
     └── vms
     2 directories, 3 files
 
+- `fedora.iso`: the latest Fedora Desktop ISO
```

Comment:
Is this change specific to 2.10 and onwards ?

Reply:
2.10 and later.